sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -401,6 +427,46 @@ object ScalaJSPluginInternal {
               "are running a JVM REPL. JavaScript things won't work.")
       )),
 
+      scalaJSNativeLibraries := {
+        collectFromClasspath(fullClasspath.value,
+            "*.js", collectJar = _.jsFiles,
+            collectFile = FileVirtualJSFile.relative)
+      },
+
+      jsDependencyManifests in jsManifestFilter := {
+        collectFromClasspath(fullClasspath.value,
+            new ExactFilter(JSDependencyManifest.ManifestFileName),
+            collectJar = _.jsDependencyManifests,
+            collectFile = { (file, _) =>
+              fromJSON[JSDependencyManifest](readJSON(IO.read(file)))
+            })
+      },
+
+      jsDependencyManifests := {
+        val filter = jsManifestFilter.value
+        val rawManifests = (jsDependencyManifests in jsManifestFilter).value
```

Comment:
> Is there any reason not to "inline" the body of jsDependencyManifests in jsManifestFilter in jsDependencyManifests?

I did want the raw manifests to be available if necessary. But maybe that is overkill.

How about this:

- Leave `jsManifestFilter` as is.
- Consolidate `jsDependencyManifests` in a single setting that filters.

?